ci-operator/config/3scale-qe/3scale-deploy/3scale-qe-3scale-deploy-main__3scale-amp-ocp4.13-lp-interop.yaml

```diff
@@ -43,17 +43,18 @@ tests:
         {
           "failure_rules":
             [
-              {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster"},
-              {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators"},
-              {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other"},
+              {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["!default"]},
+              {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default"]},
+              {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
```

Comment:
Should we add the z-stream test labels to these steps

Reply:
Yes good point, I've added the label "z-stream" to default vars using the variable FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS